crowbar_framework/config/crowbar.env

```diff
@@ -1,4 +1,6 @@
-export CROWBAR_VERSION=1.10
+export RAINBOWS_WORKERS=5
```

Comment:
@tboerger does the `RAINBOWS_WORKERS` env influence only development or also production?